backend/config/initializers/rails_admin.rb

```diff
@@ -0,0 +1,29 @@
+RailsAdmin.config do |config|
+
+  config.authorize_with do
+    authenticate_or_request_with_http_basic('Admin Authentication') do |username, password|
+      admin = Client.find_by_id(1)
+      username == admin.name.downcase && admin.authenticate(password)
```

Comment:
This might be trivial, but I think you can also do this...

```ruby
authenticate_or_request_with_http_basic('Admin Authentication') do |name, password|
  Client.find_by(name: name.downcase).try(:authenticate, password)
end

```